src/lib.rs

```diff
@@ -59,12 +59,14 @@ impl Bencher {
     ///
     /// # Timing loop
     ///
-    /// ``` ignore
-    /// start = Instant::new();
+    /// ```rust,no_run
```

Comment:
What does the `rust` do here (and in all the other occurrences below)? I haven't used it before.